### PR TITLE
Extend and improve readme

### PR DESCRIPTION
**Pull request by OndraM**
To have everything nicely prepared for 1.8.0.

Any suggestions what could be missing in the readme?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,55 +36,120 @@ Then install the library:
...
+Make sure to have compatible version of Geckodriver and Firefox!
 
-The `chromedriver` binary must be placed in system `PATH` directory, otherwise you must provide the path when starting Selenium server
-(eg. `java -Dwebdriver.chrome.driver="/path/to/chromedriver" -jar selenium-server-standalone-#.jar`).
+Run `geckodriver` binary (it start to listen on port 4444 by default):
 
-```php
-$driver = RemoteWebDriver::create($host, DesiredCapabilities::chrome());
+```sh
+geckodriver
```
Comment (williamdes) [inline README.md]: You specified the default port for chromedriver can you do it for geckodriver?

File: README.md [hunk 2]
```diff
@@ -36,55 +36,120 @@ Then install the library:
...
 
-##### Launch Firefox
+#### c) Selenium standalone server
 
-Install latest Firefox and [Geckodriver](https://github.com/mozilla/geckodriver/releases).
+[Selenium server](https://selenium.dev/downloads/) is useful especially when you need to execute multiple tests at once
+or your tests are run in different browsers - like on your CI server.
+
+Selenium server receives commands and starts new sessions using browser drivers acting like hub distributing the commands
+amongst multiple nodes.
```
Comment (williamdes) [inline README.md]: Typo?

File: README.md (hunk 1 above)
Comment (OndraM) [reply README.md]: Geckodriver has 4444 as the default (while chromedriver does not).

File: README.md [hunk 3]
```diff
@@ -36,55 +36,120 @@ Then install the library:
...
+// selenium-server-standalone-#.jar (version 2.x or 3.x)
+$host = 'http://localhost:4444/wd/hub';
+// selenium-server-standalone-#.jar (version 4.x)
+$host = 'http://localhost:4444';
+```
+
+Now you can start browser of your choice:
 
 ```php
+// Chrome
```
Comment (williamdes) [inline README.md]: Could we document the use statements so users can copy paste them?

File: README.md (hunk 1 above)
Comment (williamdes) [reply README.md]: Okay

File: README.md (hunk 3 above)
Comment (OndraM) [reply README.md]: Added above

**Pull request merged**